.github/workflows/test-java-jdbc.yml

```diff
@@ -17,6 +17,10 @@ on:
   # Allow job to be triggered manually.
   workflow_dispatch:
 
+  # Run job each night after CrateDB nightly has been published.
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Afaik the nightly tarball will run at 00:02 and takes ~3min. So this looks too tight, we should run it later. (I see no reason to let it run even an hour later to be safe)

Reply:
Apparently, there is no way to set the time zone anyway. Is it UTC then?

- https://github.com/actions/runner/issues/1177
- https://github.com/actions/runner/issues/1423
- https://github.com/orgs/community/discussions/13454

What do you recommend to use here? Currently, it means "each night at 3 o'clock".